Examples/NoModules/Chapter 04/Ex4_01A.cpp

```diff
@@ -0,0 +1,20 @@
+// Comparing data values (output using std::format())
+#include <iostream>
+#include <format>
+
+int main()
+{
+  char first {};      // Stores the first character
+  char second {};     // Stores the second character
+
+  std::cout << "Enter a character: ";
+  std::cin >> first;
+
+  std::cout << "Enter a second character: ";
+  std::cin >> second;
+
+  std::cout << std::format("The value of the expression {} < {} is {}\n",
+                           first, second, first < second);
+  std::cout << std::format("The value of the expression {} == {} is {}\n",
+                           first, second, first < second);
```

Comment:
oops!
first < second
moet zijn:
first == second
ook foutje in de text ... gemist tijdens review :(